suffuse/inode_gen.go

```diff
@@ -0,0 +1,38 @@
+package suffuse
+
+type InodeGen struct {
+  fresh chan InodeNum
+}
+
+func NewRootInode(startInode uint64)*Inode {
+  ichan := make(chan InodeNum)
+  // Monotonically increments the counter without any
+  // effort to track or reuse.
+  go func() {
+    count := uint64(startInode)
+    for {
+      ichan <- InodeNum(count)
+      count += 1
+    }
+  }()
+
+  gen := InodeGen { ichan }
+  return gen.NextDir()
+}
+
+func (x *InodeGen) Next(tp InodeType) *Inode {
+  ino := Inode { InodeGen: x, AttrMap: AttrMap{} }
+  ino.SetAttr(InodeNumKey, <- x.fresh)
+  ino.SetAttr(InodeTypeKey, tp)
+  ino.SetAttr(PermBitsKey, BasePermBits())
```

Comment:
Is there any specific reason you do not initialize `AttrMap` above with these values?